.github/workflows/release.yml

```diff
@@ -13,6 +13,11 @@ jobs:
       - name: Checkout project
         uses: actions/checkout@v2
 
+      - name: Clean previous docs
+        run: |
+          rm -rf ./src/stdlib/*/
+          rm -rf ./src/stdlib/*.md
```

Comment:
Can we make this a package.json script instead? This may require rimraf or something of the like.

Reply:
I don't know if this makes a lot of sense, just because we don't have any match scripts to generate the stdlib docs in the `package.json` we would only have `cleanStdlibDocs` or something.

(This will also only be around for a little while until we merge v2)